src/Traits/RendersImages.php

```diff
@@ -178,4 +178,45 @@ protected function renderBackground(): void
             $filledRows++;
         }
     }
+
+    protected function renderBackgroundURL(): void
+    {
+        $panel = $this->manager->read(file_get_contents($this->backgroundURL));
```

Comment:
This trait has moved and is now used by the `Layout` (see 
`src/Layout/AbstractLayout.php`) rather than the `Image`.

Note that some properties of the `Image` class have moved to the `Layout` while the ones that haven't are now accessible via the Layout's `$config` property.

Your `backgroundUrl` property should remain on the `Image` class, but you will need to update it here to reference it through the `$config` property:

```suggestion
    protected function renderBackgroundURL(): void
    {
        $panel = $this->manager->read(file_get_contents($this->config->backgroundUrl));
```